content/blog/200-million-downloads/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: AdoptOpenJDK reaches 200 million downloads
+date: "2020-10-19T09:00:00+00:00"
+author: georgeadams
+description: Today the AdoptOpenJDK project celebrates another huge milestone.
+featuredImage: "./downloads_counter.png"
+tags:
+  - AdoptOpenJDK
+  - Downloads
+---
+
+![Download Counter](./downloads_counter.png)
```

Comment:
That one looks mushy on my 4K display.

Reply:
I added images in higher resolution as a comment to this PR. @gdams must change them

Reply:
@aahlenst updated the images